### ISSUE TITLE
https://dev.gitblit.com/ certificate expired

### ISSUE THREAD

**Issue by RedShift1** [gitblit/gitblit#996]
![dev gitblit certificate](https://cloud.githubusercontent.com/assets/3098010/12182174/e050525e-b586-11e5-9671-b314e5477b5e.png)

**Comment by sschuberth** [comment]
Maybe get the new one via [Let's Encrypt](https://letsencrypt.org/)?

**Comment by gitblit** [comment]
Yes, I let it expire.  The free CA is interesting.  I'll look into that.

**Comment by tomposmiko** [comment]
@gitblit 
Can I help in this?
I would be happy to take over the server and other maintenance tasks from you.
